src/module-version-added.json

```diff
@@ -0,0 +1,100 @@
+{
```

Comment:
i just pulled this from the old repo but i'll actually commit the most up-to-date version of this.